docs/quickstart.md

```diff
@@ -0,0 +1,228 @@
+BGC file `xxxx.region001.gbk`, the BGC id is `xxxx.region001`.
+
+The spectrum id is same as the scan number in the `spectra.mgf` file in the `gnps` directory, 
+for example, given a spectrum in the mgf file with a scan `SCANS=1`, the spectrum id is `1`. 
+
+If you labelled the mzXML files (input for GNPS) with the strain id, you may need the function [extract_mappings_ms_filename_spectrum_id][nplinker.metabolomics.utils.extract_mappings_ms_filename_spectrum_id] 
+to extract the mappings from mzXML files to the spectrum ids.
+
+For the `local` mode, you need to create this file manually and put it in the working directory.
+It takes some effort to prepare this file manually, especially when you have a large number of strains.
+
+
+## 4. Prepare config file
+
+The configuration file `nplinker.toml` is required by NPLinker to specify the working directory, mode,
+and other settings for the run of NPLinker.
+
+
+The details of all settings can be found at this page [Config File](./concepts/config_file.md).
+
+To keep it simple, [default settings](./concepts/config_file.md#default-configurations) will be used automatically by NPLinker if you don't set them in your `nplinker.toml` config file.
+
+What you need to do is to set the `root_dir` and `mode` in the `nplinker.toml` file.
+
+=== "`local` mode"
+    ```toml title="nplinker.toml"
+    root_dir = "absolute/path/to/working/directory" # (1)!
+    mode = "local"
+    # and other settings you want to override the default settings 
+    ```
+
+    1. Replace `absolute/path/to/working/directory` with the **absolute** path to the working directory
+       created in step 2.
+
+
+=== "`podp` mode"
+    ```toml title="nplinker.toml"
+    root_dir = "absolute/path/to/working/directory" # (1)!
+    mode = "podp"
+    podp_id = "podp_id" # (2)!
+    # and other settings you want to override the default settings 
+    ```
+
+    1. Replace `absolute/path/to/working/directory` with the **absolute** path to the working directory
+       created in step 2.
+    2. Replace `podp_id` with the identifier of the dataset in the Paired Omics Data Platform (PODP).
+
+When you have prepared the `nplinker.toml` file, put it in the working directory.
+
+
+## 5. Run NPLinker
+
+Before running NPLinker, make sure your working directory containing input data and config file
+has the correct directory structure and names described in the [Working Directory Structure](./concepts/working_dir_structure.md) page.
+
+```python title="Run NPLinker in your working directory"
+from nplinker.nplinker import NPLinker
+
+# create an instance of NPLinker
+npl = NPLinker()
+
+# load data
+npl.load_data()
+
+# check loaded data
+print(npl.bgcs)
+print(npl.gcfs)
+print(npl.spectra)
+print(npl.molfams)
+print(npl.strains)
+
+# get the links generated by metcalf scoring
+npl.get_links(input_objects=npl.gcfs, scoring_method="metcalf")
+```
+
+You need to run NPlinker in the working directory where the `nplinker.toml` file is located, and
```

Comment:
Similarly, I'd move this information at the beginning of `5. Run NPLinker`. As a rule of thumb, I find it easier if I immediately know which files I need and where they need to be placed, and then I look at the code for example/quick implementation.

Reply:
Done